Makefile

```diff
@@ -20,8 +20,11 @@ DEST = website
 SPHINX_DEPENDENCIES = $(foreach lang, $(LANGS), $(lang)/Makefile)
 
 # Copy-paste the English Makefile everywhere it's needed (if non existing).
+en/Makefile: Makefile
+	cp $< $@
+
 %/Makefile: en/Makefile
-	cp -n $< $@
+	cp $< $@
```

Comment:
This seems to cause the sphinx build error. Is it necessary to overwrite the Makefile for all lang?